### PR TITLE
Remove dependency from application to infrastructure

### PR DESCRIPTION
**Pull request by kristianschneider**
This PR refactors the code so that the app layer does not know the infrastructure layer

### CODE REVIEW

**Comment by ZavezX** [comment]
There is still a reference to protobuf in `CoordinatorFunction` project:
devenv_9G9pA1VdFL

**Comment by ZavezX** [comment]
I see there is a reference to project `DatabaseMigration` in project `Application`. Why is this here? It seems like it shouldn't?

devenv_ceFHCm7wuX

**Comment by ZavezX** [comment]
This line can be removed as it's not used:

https://github.com/Energinet-DataHub/geh-aggregations/blob/4cb7873fbe86ab6c7428dc96f0c72686f70a7ae9/source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/CoordinatorService.cs#L17

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step01ExchangePerNeighbourStrategy.cs [hunk 1]
```diff
@@ -15,12 +15,10 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step01ExchangePerNeighbourStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step02ExchangeStrategy.cs [hunk 2]
```diff
@@ -15,14 +15,12 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step02ExchangeStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step02ExchangeStrategy.cs [hunk 3]
```diff
@@ -15,14 +15,12 @@
...
 using System.Collections.Generic;
 using System.Linq;
+using GreenEnergyHub.Aggregation.Application.Coordinator.Interfaces;
 using GreenEnergyHub.Aggregation.Application.Services;
 using GreenEnergyHub.Aggregation.Domain;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step02ExchangeStrategy.cs:20]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step03HourlyConsumptionStrategy.cs [hunk 4]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step03HourlyConsumptionStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step0809CombinedGridLossStrategy.cs [hunk 5]
```diff
@@ -15,13 +15,10 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step0809CombinedGridLossStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step10FlexConsumptionStrategy.cs [hunk 6]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step10FlexConsumptionStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step11ProductionStrategy.cs [hunk 7]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step11ProductionStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step12ProductionPerSupplierStrategy.cs [hunk 8]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step12ProductionPerSupplierStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step13HourlyConsumptionPerSupplierStrategy.cs [hunk 9]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step13HourlyConsumptionPerSupplierStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step14FlexConsumptionPerSupplierStrategy.cs [hunk 10]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step14FlexConsumptionPerSupplierStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step15ProductionPerBrpStrategy.cs [hunk 11]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step15ProductionPerBrpStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step16HourlyConsumptionPerBrpStrategy.cs [hunk 12]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step16HourlyConsumptionPerBrpStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step17FlexConsumptionPerBrpStrategy.cs [hunk 13]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step17FlexConsumptionPerBrpStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step18ProductionPerGridAreaStrategy.cs [hunk 14]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step18ProductionPerGridAreaStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step19HourlyConsumptionPerGridAreaStrategy.cs [hunk 15]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step19HourlyConsumptionPerGridAreaStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step20FlexConsumptionPerGridAreaStrategy.cs [hunk 16]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step20FlexConsumptionPerGridAreaStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step21TotalConsumptionPerGridAreaStrategy.cs [hunk 17]
```diff
@@ -15,13 +15,11 @@
 using System;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/Strategies/Step21TotalConsumptionPerGridAreaStrategy.cs:15]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Application/GreenEnergyHub.Aggregation.Application.csproj [hunk 18]
```diff
@@ -28,7 +28,6 @@ limitations under the License.
   <ItemGroup>
     <ProjectReference Include="..\GreenEnergyHub.Aggregation.DatabaseMigration\GreenEnergyHub.Aggregation.DatabaseMigration.csproj" />
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Application/GreenEnergyHub.Aggregation.Application.csproj:29]: Why is this reference in the application project?

File: source/coordinator/GreenEnergyHub.Aggregation.Infrastructure/ServiceBusProtobuf/MeteringPointMessageToDtoMapper.cs [hunk 19]
```diff
@@ -14,7 +14,11 @@
 
 using System;
 using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
 using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Infrastructure/ServiceBusProtobuf/MeteringPointMessageToDtoMapper.cs:19]: Unused using.
```suggestion
```

File: source/coordinator/GreenEnergyHub.Aggregation.Infrastructure/ServiceBusProtobuf/TimeSeriesDispatcher.cs [hunk 20]
```diff
@@ -12,14 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using GreenEnergyHub.Aggregation.Application.Coordinator.Interfaces;
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Infrastructure/ServiceBusProtobuf/TimeSeriesDispatcher.cs:15]: Unused using.
```suggestion
```

**Comment by ZavezX** [comment]
Unused using can be removed:
https://github.com/Energinet-DataHub/geh-aggregations/blob/4cb7873fbe86ab6c7428dc96f0c72686f70a7ae9/source/coordinator/GreenEnergyHub.Aggregation.Tests/ConsumptionStrategiesTests.cs#L23

**Comment by ZavezX** [comment]
Unused using can be removed:
https://github.com/Energinet-DataHub/geh-aggregations/blob/4cb7873fbe86ab6c7428dc96f0c72686f70a7ae9/source/coordinator/GreenEnergyHub.Aggregation.Tests/InputProcessorTests.cs#L24

File: source/messaging/source/GreenEnergyHub.Messaging.Integration.ServiceCollection/HandlerExtensions.cs [hunk 21]
```diff
@@ -15,6 +15,8 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging.Integration.ServiceCollection/HandlerExtensions.cs:19]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging.Protobuf/ProtobufMessageSerializer.cs [hunk 22]
```diff
@@ -16,6 +16,9 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Google.Protobuf;
+using GreenEnergyHub.Aggregation.Application.Coordinator.Interfaces;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging.Protobuf/ProtobufMessageSerializer.cs:19]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging.Protobuf/ProtobufMessageSerializer.cs [hunk 23]
```diff
@@ -16,6 +16,9 @@
...
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging.Protobuf/ProtobufMessageSerializer.cs:21]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging.Protobuf/ProtobufOutboundMapper.cs [hunk 24]
```diff
@@ -13,6 +13,9 @@
 // limitations under the License.
 
 using Google.Protobuf;
+using GreenEnergyHub.Aggregation.Application.Coordinator.Interfaces;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging.Protobuf/ProtobufOutboundMapper.cs:16]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging.Protobuf/ProtobufOutboundMapper.cs [hunk 25]
```diff
@@ -13,6 +13,9 @@
...
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
 using GreenEnergyHub.Messaging.Transport;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging.Protobuf/ProtobufOutboundMapper.cs:19]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging.Protobuf/ProtobufOutboundMapperFactory.cs [hunk 26]
```diff
@@ -13,7 +13,8 @@
 // limitations under the License.
 
 using System;
-using GreenEnergyHub.Messaging.Transport;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging.Protobuf/ProtobufOutboundMapperFactory.cs:17]: Unused using.
```suggestion
```

**Comment by ZavezX** [comment]
There are two files that seem a little odd to me as it looks like they shouldn't be there:
https://github.com/Energinet-DataHub/geh-aggregations/blob/4cb7873fbe86ab6c7428dc96f0c72686f70a7ae9/source/messaging/source/GreenEnergyHub.Messaging.Protobuf/ProtobufOutboundMapper%601.cs
https://github.com/Energinet-DataHub/geh-aggregations/blob/4cb7873fbe86ab6c7428dc96f0c72686f70a7ae9/source/messaging/source/GreenEnergyHub.Messaging.Protobuf/ProtobufInboundMapper%601.cs

File: source/messaging/source/GreenEnergyHub.Messaging/Dispatching/BaseCommandHandler.cs [hunk 27]
```diff
@@ -14,6 +14,8 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Dispatching/BaseCommandHandler.cs:18]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Dispatching/BaseRequestHandler.cs [hunk 28]
```diff
@@ -13,6 +13,8 @@
 // limitations under the License.
 using System.Threading;
 using System.Threading.Tasks;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Dispatching/BaseRequestHandler.cs:17]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Dispatching/HubCommand.cs [hunk 29]
```diff
@@ -12,9 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
- using MediatR;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Dispatching/HubCommand.cs:16]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Dispatching/HubCommandHandler.cs [hunk 30]
```diff
@@ -15,6 +15,8 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Dispatching/HubCommandHandler.cs:19]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Dispatching/HubCommandMediator.cs [hunk 31]
```diff
@@ -14,6 +14,8 @@
 
  using System;
  using System.Threading.Tasks;
+ using GreenEnergyHub.Aggregation.Domain;
+ using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Dispatching/HubCommandMediator.cs:18]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Dispatching/HubRequest.cs [hunk 32]
```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Dispatching/HubRequest.cs:16]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Dispatching/HubRequestHandler.cs [hunk 33]
```diff
@@ -15,6 +15,8 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Dispatching/HubRequestHandler.cs:19]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Dispatching/HubRequestMediator.cs [hunk 34]
```diff
@@ -14,6 +14,8 @@
 
 using System;
 using System.Threading.Tasks;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Dispatching/HubRequestMediator.cs:18]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Dispatching/IHubCommandMediator.cs [hunk 35]
```diff
@@ -13,6 +13,8 @@
 // limitations under the License.
 
 using System.Threading.Tasks;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Dispatching/IHubCommandMediator.cs:17]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Dispatching/IHubRequestBulkMediator.cs [hunk 36]
```diff
@@ -14,6 +14,8 @@
 
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Dispatching/IHubRequestBulkMediator.cs:18]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Dispatching/IHubRequestMediator.cs [hunk 37]
```diff
@@ -13,6 +13,8 @@
 // limitations under the License.
 
 using System.Threading.Tasks;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Dispatching/IHubRequestMediator.cs:17]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/IHubRehydrate.cs [hunk 38]
```diff
@@ -15,6 +15,8 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Threading.Tasks;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/IHubRehydrate.cs:19]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/JsonSerializerWithOption.cs [hunk 39]
```diff
@@ -16,10 +16,11 @@
 using System.Text.Json;
 using System.Threading;
 using System.Threading.Tasks;
+using GreenEnergyHub.Aggregation.Application.Coordinator.Interfaces;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/JsonSerializerWithOption.cs:19]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Transport/IInboundMessage.cs [hunk 40]
```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Transport/IInboundMessage.cs:16]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Transport/MessageDispatcher.cs [hunk 41]
```diff
@@ -16,36 +16,43 @@
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
+using GreenEnergyHub.Aggregation.Application.Coordinator.Interfaces;
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Transport/MessageDispatcher.cs:21]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Transport/MessageSerializer.cs [hunk 42]
```diff
@@ -14,6 +14,9 @@
 
 using System.Threading;
 using System.Threading.Tasks;
+using GreenEnergyHub.Aggregation.Application.Coordinator.Interfaces;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Transport/MessageSerializer.cs:17]: Unused using.
```suggestion
```

File: source/messaging/source/GreenEnergyHub.Messaging/Transport/MessageSerializer.cs [hunk 43]
```diff
@@ -14,6 +14,9 @@
...
+using GreenEnergyHub.Aggregation.Domain;
+using GreenEnergyHub.Aggregation.Domain.DTOs;
```
Comment (ZavezX) [inline source/messaging/source/GreenEnergyHub.Messaging/Transport/MessageSerializer.cs:19]: Unused using.
```suggestion
```

**Review by ZavezX — CHANGES_REQUESTED** [review]
Very nice work on this! Just a lot of unused usings and some questions along the way.

**Comment by kristianschneider** [comment]
> There is still a reference to protobuf in `CoordinatorFunction` project:
> 
> devenv_9G9pA1VdFL

It is required for DI of 
`            builder.Services.SendProtobuf<Document>();
`
As far as I understand it's ok for the executing part to know infrastructure

**Comment by kristianschneider** [comment]
> I see there is a reference to project `DatabaseMigration` in project `Application`. Why is this here? It seems like it shouldn't?
> 
> devenv_ceFHCm7wuX

It is used to package the migration dll into the application artifact. It is to prevent having to create an entire deployment pipeline just for the database migration. Personally I don't have a nit with it. But if we find that this should be the case I suggest we create a story for it

File: source/coordinator/GreenEnergyHub.Aggregation.Application/GreenEnergyHub.Aggregation.Application.csproj (hunk 18 above)
Comment (kristianschneider) [reply source/coordinator/GreenEnergyHub.Aggregation.Application/GreenEnergyHub.Aggregation.Application.csproj:29]: Se my comment below

**Review by ZavezX — APPROVED** [review]

**Comment by ZavezX** [comment]
@kristianschneider Just a few conflicts and this looks good!

**Review by jonasdmoeller — APPROVED** [review]

**Pull request merged**